### ISSUE TITLE
ERROR - ws_thread - Couldn't connect to WS! Exiting.

### ISSUE THREAD

**Issue by NichJohn83** [Abhisheksoni1/BitmexBot#1]
`fatal: No names found, cannot describe anything.
2018-03-08 18:34:03,389 - INFO - bitmex_bot - BitMEX bot Version: v1.1

2018-03-08 18:34:03,389 - INFO - ws_thread - Connecting to wss://testnet.bitmex.com/realtime?subscribe=quote:XBTUSD,trade:XBTUSD,instrument,order:XBTUSD,execution:XBTUSD,margin,position
2018-03-08 18:34:03,390 - INFO - ws_thread - Authenticating with API Key.
2018-03-08 18:34:03,391 - INFO - ws_thread - Started thread
2018-03-08 18:34:08,401 - ERROR - ws_thread - Couldn't connect to WS! Exiting.`

@Abhisheksoni1 

Any clue as to why I'm getting this error?

**Comment by Abhisheksoni1** [comment]
@NichJohn83 yeah you have to place your key and secret in the bot setting.py file
